help/grandorgue.xml

```diff
@@ -9913,9 +9913,22 @@ on which the pipes of the are placed.
           <term>Percussive</term>
           <listitem>
             <para>
-(boolean, required) If false, the samples are played as is (without
-any loop/release handling)
-     </para>
+	      (boolean, required) If false, the samples are played as is
+	      (without any loop/release handling).
+	    </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>HasIndependentRelease</term>
+          <listitem>
+            <para>
+	      (boolean, default false or inherited from the Organ, Windchest or
+	      Rank). If true than release is played independently on the attack.
```

Comment:
If true, releases are played independently of the attacks.

Both because this applies to multiple pipes and also since there can be more than one attack/release each pipe.

Reply:
Fixed